libsvm-kitware/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd TPL/libsvm-3.1-custom
+
+make lib
+mkdir $PREFIX/lib
+mv libsvm* $PREFIX/lib/
```

Comment:
It appears these files are not in the v0.1 tag that I currently have as the git revision.  That was the latest tag I could find, aside from some sort of "backup tag" before some refactoring.  Can you tag a new release of SMQTK?  It's highly frowned on to have conda packages built against untagged code.

Also, it appears that these files are part of SMQTK, not the libsvm folder within that repo.  Should they move into libsvm, or should they be packaged with SMQTK?